common/src/Microsoft.Azure.IIoT.Agent.Framework/src/Agent/Default/WorkerSupervisor.cs

```diff
@@ -77,11 +77,11 @@ public class WorkerSupervisor : IWorkerSupervisor, IDisposable {
         }
 
         /// <summary>
-        /// Stop worker
+        /// Stop a single worker
         /// </summary>
         /// <returns>awaitable task</returns>
         private async Task StopWorker() {
-            // sort worker, so that worker in state stopped, stoping or WaitingForJob will terminate first 
+            // sort workers, so that a worker in state Stopped, Stopping or WaitingForJob will terminated first 
```

Comment:
terminate

Reply:
In German, we call it "verschlimbessern"  :) changed the text